interpreter/k8s/src/main/scala/io/buoyant/interpreter/k8s/ConfigMapInterpreterInitializer.scala

```diff
@@ -0,0 +1,79 @@
+package io.buoyant.interpreter
+package k8s
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.Dtab
+import com.twitter.finagle.Stack.Params
+import com.twitter.finagle.naming.NameInterpreter
+import com.twitter.finagle.param.Label
+import com.twitter.logging.Logger
+import com.twitter.util._
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.ClientConfig
+import io.buoyant.k8s.v1._
+import io.buoyant.namer.{ConfiguredDtabNamer, InterpreterConfig, InterpreterInitializer, Param}
+import java.util.concurrent.atomic.AtomicReference
+
+class ConfigMapInterpreterInitializer extends InterpreterInitializer {
+  val configClass = classOf[ConfigMapInterpreterConfig]
+  override def configId: String = "io.l5d.k8s.configMap"
+}
+
+object ConfigMapInterpreterInitializer extends ConfigMapInterpreterInitializer
+
+case class ConfigMapInterpreterConfig(
+  host: Option[String],
+  port: Option[Port],
+  namespace: Option[String],
+  name: String,
```

Comment:
I believe that jackson will happily parse a config that is missing these properties and fill them with nulls.  We typically add an assert to make sure they're non-null.